include/crocoddyl/multibody/contacts/contact-1d.hxx

```diff
@@ -90,9 +100,23 @@ void ContactModel1DTpl<Scalar>::updateForce(const boost::shared_ptr<ContactDataA
                  << "lambda has wrong dimension (it should be 1)");
   }
   Data* d = static_cast<Data*>(data.get());
-  const Eigen::Ref<const Matrix3s> R = d->jMf.rotation();
-  data->f.linear() = R.col(2) * force[0];
-  data->f.angular() = d->jMf.translation().cross(data->f.linear());
+  pinocchio::SE3 jMc;
```

Comment:
Let's define a global variable for this, and called it `jMc_`.
This avoids to allocate data dynamically.